keys/hd/hdpath_test.go

```diff
@@ -0,0 +1,73 @@
+package hd
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/tendermint/go-crypto/keys/bip39"
+)
+
+func ExampleStringifyPathParams() {
+	path := NewParams(44, 0, 0, false, 0)
+	fmt.Println(path.String())
+	// Output: 44'/0'/0'/0/0
+}
+
+func ExampleSomeBIP32TestVecs() {
```

Comment:
Why do these print instead of checking whether or not the outputs are correct in the testcase?

Reply:
This prints do exactly that: https://blog.golang.org/examples